WcaOnRails/app/assets/stylesheets/competitions.scss

```diff
@@ -167,11 +165,20 @@ $venue-map-wrapper-height: 400px;
     width: 100px;
   }
 
-  #submit-buttons {
-    display: block;
+  #state {
+    min-width: 220px;
+    width: 100%;
+  }
+
+  #display {
     width: 100%;
     margin-top: 20px;
-    text-align: center;
+    .btn-group {
```

Comment:
I believe we can get rid of all this css and just use .btn-group-justified. See http://getbootstrap.com/components/#btn-groups-justified